example/SliderExample.js

```diff
@@ -223,4 +223,17 @@ export const examples = [
       return <SliderExample value={0.6} vertical />;
     },
   },
+  {
+    title: 'Disabled slider',
+    platform: 'android',
+    render(): Element<any> {
+      return (
+        <SliderExample
+          disabled
+          accessibilityState={{disabled: false}}
```

Comment:
How about creating two separate examples: one with `disabled` prop, another for `accessibilityState`.
The goal of this PR is to avoid conflicting disabled states, so it would be better to show that both these properties can be used separately without fear of another.
I'm afraid this example will cause confusions as it could suggest that both these has to be set to equal value to make it work.

Reply:
thanks a lot for the review. I made the change with commit https://github.com/callstack/react-native-slider/pull/354/commits/8a8436e51f0f5a07b130b18212d17d0d8a71ae85. Sorry for the delay, but I was very busy.